test/integration/test-submit-custom-header.js

```diff
@@ -31,17 +44,15 @@ var server = http.createServer(function(req, res) {
 
   form
     .on('field', function(name, value) {
-      fieldsPassed--;
-      var field = FIELDS[name];
-      assert.ok(field);
-      assert.strictEqual(value, ''+field);
+      var field = FIELDS.shift();
+      assert.strictEqual(name, field.name);
+      assert.strictEqual(value, field.value+'');
     })
     .on('file', function(name, file) {
-      fieldsPassed--;
-      var field = FIELDS[name];
-      assert.ok(field);
-      assert.strictEqual(file.name, path.basename(field.path));
-      assert.strictEqual(file.type, mime.lookup(file.name));
+      var field = FIELDS.shift();
```

Comment:
I'm one of the maintainers, so issues here would be better way to discuss library specific questions. Although for faster turnaround, you can find me on gtalk (iam@alexindigo.com).

Actually I'm thinking maybe it's time to create some kind of general chat room. Like I noticed Electron has it's own Slack org.